com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md

```diff
@@ -42,6 +42,8 @@ From Unity 2020.2, the light layer properties have move from the HDRP settings t
 
 From Unity 2020.2, in physically based sky, the sun disk intensity is proportional to its size. Before this the sun disk was incorrectly not drive by the sun disk size.
 
+From Unity 2020.2, EmissiveColorLDR property when using the UseEmissiveIntensity property is not correctly handle as sRGB color space in Decal, Lit, LayeredLit and Unlit shader. Previously the EmissiveColor was handled as linear RGB color space. A migration script handle the conversion automatically to keep visual matching.
```

Comment:
From Unity 2020.2, if you previously used the **UseEmissiveIntensity** property in either a Decal, Lit, LayeredLit, or Unlit Material, be aware that the **EmissiveColorLDR** property is in sRGB color space. Previously HDRP handled **EmissiveColor** as being in linear RGB color space so there may be a mismatch in visuals between versions. To fix this mismatch, HDRP includes a migration script that handles the conversion automatically.